### DISCUSSION TITLE
Is Polkadot now fully supported in WalletConnect v2.0 ?

### DISCUSSION THREAD

**Discussion by Apocyz** [WalletConnect/walletconnect-monorepo#1615] [Developer Support] [answered]
Hello, 

I am trying to know if Polkadot/Kusama chains are now supported by WalletConnect v2 or if it's still in progress. 
I was not able to get information from the RPC documentation : https://docs.walletconnect.com/2.0/advanced/rpc-reference/ethereum-rpc or the explorer : https://explorer.walletconnect.com/?chains=solana%3A4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ

Thanks

**Comment by chadyj** [comment, marked as answer]
Yes Polkadot is supported.

You can see working examples in our dapp and wallet sample apps

https://react-app.walletconnect.com/
https://react-wallet.walletconnect.com/

**Reply by Apocyz** [reply]
Thanks a lot for the timely answer ;)
